drydock/templates/drydock/k8s/patches/add-image-pull-secret.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
If you use the `kind` field in the patch file with the `Deployment` value, would this work to patch a cronjob? Be careful with the resources structure; in a Deployment, the `imagePullSecrets` field is obtained with  `spec.template.spec.imagePullSecrets` but in a cronJob, it is located at `spec.jobTemplate.spec.template.spec.imagePullSecrets`